test/LocalNeighborhood/test_ln.py

```diff
@@ -44,3 +45,31 @@ def test_format_error(self):
                                output_file=OUT_FILE)
 
     # Write tests for the Local Neighborhood run function here
+    def test_run_require(self):
+        out_path = Path(OUT_FILE)
+        out_path.unlink(missing_ok=True)
+        # Only include required arguments
+        LocalNeighborhood.run(
+            network_file=Path(TEST_DIR, 'input', 'ln-network.txt'),
+            nodes_file=Path(TEST_DIR, 'input', 'ln-nodes.txt'),
+            output_file=OUT_FILE
+        )
+        assert out_path.exists()
+
+    def test_run_missing(self):
```

Comment:
This test will check the behavior when a required input file is not provided. You can provide a correct network file but omit the nodes file. That should throw an error, so you will have to tell pytest to expect an error in order for the test to pass. Check https://github.com/zhiqianx/spras/blob/master/test/PathLinker/test_pathlinker.py#L38 for an example.